notebooks/person-counting-webcam/person-counting.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>please add -q flag (for reducing installation logs) and --extra-index-url <a href="https://download.pytorch.org/whl/cpu" target="_blank">https://download.pytorch.org/whl/cpu</a> (for reducing pytorch package size if it is not installed)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>